### PR TITLE
Add esmlab recipe

### PR DESCRIPTION
**Pull request by andersy005**

### CODE REVIEW

**Comment by andersy005** [comment]
@kmpaul, I added you on the list of recipe-maintainers. Are you OK with being a maintainer?

**Comment by kmpaul** [comment]
Yes.  That's fine.

**Comment by andersy005** [comment]
@conda-forge/help-python, this recipe is ready for review.

File: recipes/esmlab/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  
+
+build: 
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  noarch: python
+  number: 0
+
+requirements:
+
+  build:
```
Comment (beckermr) [inline recipes/esmlab/meta.yaml]: This should be `host`, not build.

**Comment by andersy005** [comment]
Thank you, @beckermr!

**Pull request merged**